.snyk

```diff
@@ -0,0 +1,16 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.25.0
+ignore: {}
+patch: {}
+exclude:
+  global:
+    - '.github/actions'
+    - 'collector/lib/'
```

Comment:
I'm not familiar with this tool, but isn't it to broad exclusion pattern?

Reply:
[snyk.io](https://snyk.io/) is a required check in RHTAP. 
To pass the pipeline initially, we decided to exclude all violations for now and come back later to address them.